ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -1,72 +1,100 @@
 #!/bin/bash
 
+manage_queue() {
+	GET_QUEUE=$(curl -k -s --resolve "$endpoint_resolve" "https://${endpoint}/queue/api/json")
+
+	QUEUE_FILTERED_JOB_NAME=$(echo $GET_QUEUE | jq --arg testname "$test_name" '.items[] | select(.task.name == $testname)')
```

Comment:
If this fails then we should exit with error.

Reply:
okay I added a error check